id,module,name,file,doc,markers,status,message,duration
functional_correctness_test_folder/post_cut-off/rag/gpt-4.1/no_docstr/test_snippet_19.py::test_case_0,functional_correctness_test_folder.post_cut-off.rag.gpt-4.1.no_docstr.test_snippet_19,test_case_0,functional_correctness_test_folder/post_cut-off/rag/gpt-4.1/no_docstr/test_snippet_19.py,,,failed,AttributeError: type object 'MarkdownCleaner' has no attribute 'NORMAL_FORMULA_CHARS',8.73330282047391e-05
functional_correctness_test_folder/post_cut-off/rag/gpt-4.1/no_docstr/test_snippet_19.py::test_case_1,functional_correctness_test_folder.post_cut-off.rag.gpt-4.1.no_docstr.test_snippet_19,test_case_1,functional_correctness_test_folder/post_cut-off/rag/gpt-4.1/no_docstr/test_snippet_19.py,,,failed,AttributeError: type object 'MarkdownCleaner' has no attribute 'NORMAL_FORMULA_CHARS',8.304195944219828e-05
functional_correctness_test_folder/post_cut-off/rag/gpt-4.1/no_docstr/test_snippet_19.py::test_case_2,functional_correctness_test_folder.post_cut-off.rag.gpt-4.1.no_docstr.test_snippet_19,test_case_2,functional_correctness_test_folder/post_cut-off/rag/gpt-4.1/no_docstr/test_snippet_19.py,,,failed,AttributeError: type object 'MarkdownCleaner' has no attribute 'NORMAL_FORMULA_CHARS',7.30000901967287e-05
